### ISSUE TITLE
Slap City (725480)

### ISSUE THREAD

**Issue by oct2pus** [ValveSoftware/Proton#23]
I am running Debian Buster, and with the steam beta version of Proton I have a variety of issues compared to running Steam in Wine Staging.

Character models do not load, all other game models have been observed to load, although due to  #25 I am struggling to test this further.

Edit: I have a Radeon RX 570, and I am running Gnome under Wayland (So Steam is running under xWayland)

**Comment by oct2pus** [comment]
This can be fixed with -force-d3d9 as a launch option, i'm not certain if this is also a windows issue.

**Comment by cjwijtmans** [comment]
Are you sure its the models or textures? see #134

**Comment by kisak-valve** [comment]
Slap City - Fullscreen Freezes Game

Issue transferred from https://github.com/ValveSoftware/Proton/issues/25.
@oct2pus posted on 2018-08-22T03:04:15:

I am running Debian Buster, and with the steam beta version of Proton I have a variety of issues compared to running Steam in Wine Staging.

I had set the game to use Fullscreen, and now with Fullscreen, the game stalls until Gnome prompts me to kill the process.

Edit: I have a Radeon RX 570, and I am running Gnome under Wayland (So Steam is running under xWayland)
***
@oct2pus commented on 2018-08-22T17:33:57

This can be fixed with `-force-d3d9` as a launch option, i'm not certain if this is also a windows issue.

**Comment by kisak-valve** [comment]
Slap City - No Stock Icons

Issue transferred from https://github.com/ValveSoftware/Proton/issues/26.
@oct2pus posted on 2018-08-22T03:05:11:

I am running Debian Buster, and this is an issue I also had running steam through wine staging, I am missing stock icons for all characters in Slap City.

Edit: I have a Radeon RX 570, and I am running Gnome under Wayland (So Steam is running under xWayland)
***
@Mikerr1111 commented on 2018-08-22T03:49:55

I think you should just add this to the list of your previous issue #25. Since it's for the same game.

**Comment by oct2pus** [comment]
Alright, glad to know

@cjwijtmans im not certain how i'd test this, but that could be it.

**Comment by oct2pus** [comment]
Editing lobby names produces a visually bugged textfield that does not display text (altough clicking a similar visually bugged submit button, it does function like expected, abet you're typing blind)

**Comment by Brisse89** [comment]
@oct2pus Could you try x.org? I think this is a driver issue because this fullscreen freeze happens to me with pretty much any Vulkan application on Debian Sid when using (x)Wayland (tested both GNOME and Sway). No problem on x.org though. AMD R9 Fury running on Mesa 18.1.9 drivers.

**Comment by oct2pus** [comment]
@Brisse89 you are right, I do still have other graphical issues like missing models/model textures, but I can run in full screen.

**Comment by Brisse89** [comment]
@oct2pus You can run in fullscreen on x.org but not on Wayland? That's what I'm seeing too and I think it worked fine on x.org 1.19 and then the issue came when updating to x.org 1.20 and it hasn't been fixed since. I don't think this is a Proton or DXVK issue, but rather a RADV issue, or perhaps even a Wayland issue.

**Comment by Ruedii** [comment]
I have a similar issue.  I don't have issues running regular or fullscreen, but many vital graphics functions just don't work in DXVK.  (I'm using RADV as well.  Rx480 in my case.)

I should probably try using WineD3D11 and get back to you.

Attached is the Unity Log for the game running with DXVK.  
[output_log.txt](https://github.com/ValveSoftware/Proton/files/2503771/output_log.txt)

Update:
Issue identical with WineD3D11

**Comment by Ruedii** [comment]
BTW, I made an entry on the forum asking the developers to help fix these issues.  I know they don't have to, but I'm sure the chance to make even minor contributions to something as substantial as Wine, through Valve no less, is a social capital opportunity that would be hard to pass up.

It would be nice for the other SteamPlay users to +1 on the post.

**Comment by kisak-valve** [comment]
Slap City - Native GC Issue

Issue transferred from https://github.com/ValveSoftware/Proton/issues/3043.
@Settyness posted on 2019-09-06T13:28:39:

Slap City - AppID 725480

This game had worked nearly flawlessly with just `-force-d3d9` defined under "Set launch options..." however, roughly six months ago, the game refused to launch under any circumstances and ProtonDB has been plagued  by "Borked" reports ever since.

This was right around the time of [this](https://steamcommunity.com/games/725480/announcements/detail/1754619776318997947) update. In a nutshell, I've found that Slap City's native GameCube controller support has somehow completely broken Proton compatibility. Running the game with `-disable-native-gc` makes the game playable once more.

**Comment by cecton** [comment]
The game runs out of the box (without launch options) nowadays and runs well on Steam Deck:

27-33 fps on the map 
50 solid in mission 
37-50 in versus with 3 cpus
